rpcd/playbooks/roles/rpc_pre_upgrade/tasks/remove_beaver.yml

```diff
@@ -0,0 +1,57 @@
+---
```

Comment:
I still think these tasks should be moved into the ``rpc-pre-upgrade.yml`` file as it's own play(not it's own playbook file). This is because of the same reason stated [here](https://github.com/rcbops/rpc-openstack/pull/1423). Right now were are iterating through all of the hosts one-by-one, which isn't very efficient.